test/recipes/30-test_evp_data/evppkey.txt

```diff
@@ -748,6 +748,40 @@ Result = KEYOP_INIT_ERROR
 Function = EVP_PKEY_verify_init
 Reason = operation not supported for this keytype
 
+Title = X448 test vectors (from RFC7748 6.2)
```

Comment:
Maybe 448/25519 should be in a separate file?

Reply:
Hmmm...not sure about that. It seems to make sense to keep it together with all of the other EVP_PKEY tests?